### ISSUE TITLE
Ask for competitor's color scheme during pre registration

### ISSUE THREAD

**Issue by jfly** [cubing/ccm#9]
This could let us show scramblers the correct color scheme.

**Comment by coder13** [comment]
Relevant still? Make this wca side? This would also slow down scrambling as you have to figure out who you're scrambling for and have ccm show the scramble in their color scheme. Easiest to just inform the scramblers of how to orient the cubes.

**Comment by jfly** [comment]
Yeah, this is something for a science fiction future where there are no scorecards, and you have to use a QR code to identify which scramble to give someone (because otherwise you don't know what scramble they're on). I've created a Science Fiction milestone for stuff like this.
